### DISCUSSION TITLE
Search Box Not Showing Up, Just a Search Icon

### DISCUSSION THREAD

**Discussion by TimMarvelHero** [valinet/ExplorerPatcher#3286] [General]
I installed the latest update of Windows 11 and when I did do that, my start menu reverted to Windows 11 and my search box was gone, it was just a search icon, and I decided to update Explorer Patcher, that fixed the issue for the start menu but it didn't fix the search box. I checked Explorer Patcher's updates again and I was on the latest version, Windows updates, also latest, I tried to restart Explorer, didn't work, I restarted Windows, didn't work.

**Comment by pyrates999** [comment]
Update to the latest pre-release of EP: https://github.com/valinet/ExplorerPatcher/releases/tag/22621.3527.65.5_6f6666b

**Reply by TimMarvelHero** [reply]
I downloaded it and it did not fix the issue unfortunately.

**Comment by pyrates999** [comment]
if the search box goes invisible after typing into it, this is a duplicate of: https://github.com/valinet/ExplorerPatcher/issues/3107

please close this as this isn't planned to be fixed due to windows 11 24h2 removing the windows 10 start menu code entirely.

**Reply by TimMarvelHero** [reply]
for me it's just a search icon rather than a search box.

**Reply by TimMarvelHero** [reply]
![image](https://github.com/valinet/ExplorerPatcher/assets/152800265/c5cb02e8-32db-4cf3-a4c1-ed934e870ca9)

**Reply by TimMarvelHero** [reply]
it was a search box before I updated. I am on 23H2 on the latest update. Not 24H2

**Reply by TimMarvelHero** [reply]
This is a different situation, in the properties section, I chose my search button to be a Search Box, but it's shows up as a Search Icon on my taskbar instead. On the previous version I was on I had the Search Box, but when I updated to the latest 23H2 update it's a Search Icon instead of a Search Box even though in the properties section I chose it to be a Search Box. Not a Search Icon. I am aware the Search Box disappears whenever I typed on it on the previous version I was on.

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.

**Reply by TimMarvelHero** [reply]
Version 23H2 (OS Build 22631.3593)

**Reply by pyrates999** [reply]
you set the taskbar to windows 10 I presume?

**Reply by TimMarvelHero** [reply]
yes

**Reply by pyrates999** [reply]
is this it? https://github.com/valinet/ExplorerPatcher/issues/3215

**Reply by TimMarvelHero** [reply]
I don't think so. My search button does work but I want it to be a search box which started appearing as an ordinary search button after I updated. Searching works fine. Just not the aesthetics.

**Reply by pyrates999** [reply]
can you right click on the taskbar, go to search and select **show search box**?

**Reply by imfromxboxandusingmicrosoftedge** [reply]
he already setted it as search box before he updated

**Reply by TimMarvelHero** [reply]
yes

**Reply by pyrates999** [reply]
lets try the following:

1. right click on the taskbar, go to search and select **show search icon**
2. right click on the taskbar, go to search and select **show search box**

Does that now show the search as a search box?

Or does it still show the search as a search icon?

**Reply by TimMarvelHero** [reply]
it still shows the search as a search icon

**Reply by pyrates999** [reply]
Ok, please wait for a developer to respond.

**Comment by Amrsatrio** [comment]
https://github.com/valinet/ExplorerPatcher/issues/3227

This is due to explorer.exe symbols being nuked from the symbol servers. A bit pain to revise the method for enabling the search box... it's also not that useful because the Windows 10 taskbar is going away.

**Reply by TimMarvelHero** [reply]
alright. But my question is will it be fixed?

**Reply by imfromxboxandusingmicrosoftedge** [reply]
we will never know

**Reply by TimMarvelHero** [reply]
I assume they fixed it

**Reply by TimMarvelHero** [reply]
![image](https://github.com/valinet/ExplorerPatcher/assets/152800265/44b9aca0-af50-405d-8096-ec5a18e89371)

**Comment by DurgeDriven** [comment]
I have latest version.
Search Icon still disappears randomly.
If I click lock all taskbars the icon returns.
Then randomly  it disappears again.

Thought it may have something to do with using Open Shell Start menu.

**Reply by pyrates999** [reply]
Please post windows version and build.  You can find this by running `winver`.  Please include if you are running x86-64 or Arm.  And what language your windows is in.
Please post explorer patcher version, by running EP properties application and click on `About` on the left side.
